File: BackEnd/WebServer/WebServer/Models/Api/Response/AuthenticationResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebServer.Models.Api.Response
{
    public class AuthenticationResponseModel
    {
        public string Username { get; set; }

        public string JWToken { get; set; }

        public string ProfileImageID { get; set; }
    }
}

File: FrontEnd/DemoApp/app/src/main/java/com/example/demoapp/data/model/api/response/AuthenticationResponseModel.java
package com.example.demoapp.data.model.api.response;

public class AuthenticationResponseModel
{
    private String username;

    private String jwToken;

    private String profileImageID;

    public AuthenticationResponseModel() {}

    public AuthenticationResponseModel(String username, String jwToken)
    {
        this.username = username;
        this.jwToken = jwToken;
    }

    public String getUsername()
    {
        return username;
    }

    public void setUsername(String username)
    {
        this.username = username;
    }

    public String getJwToken()
    {
        return jwToken;
    }

    public void setJwToken(String jwToken)
    {
        this.jwToken = jwToken;
    }

    public String getProfileImageID()
    {
        return profileImageID;
    }

    public void setProfileImageID(String profileImageID)
    {
        this.profileImageID = profileImageID;
    }
}
